src/pipeline/pipeline.ts

```diff
@@ -76,12 +81,20 @@ export type PipelineOperation = (req: Request, res: Response) => Promise<void |
 type Pipeline = PipelineOperation[];
 
 const Pipelines: Record<string, Pipeline> = {
-	query: [parseBQLQuery, buildTQLQuery, runTQLQuery, parseTQLRes, dispatchPipeline],
-	mutation: [fillBQLMutation, preQuery, parseBQLMutation, buildTQLMutation, runTQLMutation, parseTQLRes],
+	query: [enrichBQLQuery, newBuildTQLQuery, newRunTQLQuery, parseTQLQuery],
+	mutation: [
+		fillBQLMutation,
```

Comment:
This we will rename to enrichBQLMutation si it is coherent with the enrichBQLQuery, but later